core/src/main/java/com/linecorp/armeria/server/healthcheck/DefaultHealthCheckUpdateHandler.java

```diff
@@ -32,9 +32,14 @@
 import com.linecorp.armeria.common.MediaType;
 import com.linecorp.armeria.common.util.UnmodifiableFuture;
 import com.linecorp.armeria.server.HttpStatusException;
+import com.linecorp.armeria.server.Server;
 import com.linecorp.armeria.server.ServiceRequestContext;
 
-enum DefaultHealthCheckUpdateHandler implements HealthCheckUpdateHandler {
+/**
+ * Handler which updates the healthiness of the {@link Server}. Supports {@code PUT}, {@code POST} and
+ * {@code PATCH} requests and tells if the {@link Server} needs to be marked as healthy or unhealthy.
+ */
+public enum DefaultHealthCheckUpdateHandler implements HealthCheckUpdateHandler {
```

Comment:
I changed the access modifier to public in order to let users select their update handler`(`DefaultHealthCheckUpdateHandler` or `HealthStatusUpdateHandler`)

Reply:
Question) what do you think of just combining the two `HealthCheckUpdateHandler` and adding similar logic as `HealthCheckService`?

e.g.
```
if jsonNode.contains("healthy") {
  // previous logic
} else if jsonNode.contains("status") {
  // update to the status
} else {
  // 400
}
```

Reply:
Once done, please don't forget to make this class package-private again.